django_project/geosight/data/api/v1/related_table.py

```diff
@@ -0,0 +1,50 @@
+from drf_yasg.openapi import Response
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework.exceptions import MethodNotAllowed
+
+from core.api_utils import ApiTag
+from geosight.data.api.v1.base import BaseApiV1Resource
+from geosight.data.models import RelatedTable
+from geosight.data.serializer.related_table import RelatedTableApiSerializer
+
+
+class RelatedTableViewSet(BaseApiV1Resource):
+    model_class = RelatedTable
+    serializer_class = RelatedTableApiSerializer
+    extra_exclude_fields = []
+    pagination_class = None
```

Comment:
From what we discussed it looked to me we wanted to return the list of related tables directly without pagination?

Reply:
Hi @victorzinho 
I think it is best to still return in pagination?

Reply:
Ok, changed.